### PR TITLE
Test and fix for Nullable enum support

### PR DESCRIPTION
**Pull request by jezza323**

### CODE REVIEW

File: MicroLite/TypeConverters/TypeConverter.cs [hunk 1]
```diff
@@ -173,7 +173,7 @@ public static DbType ResolveDbType(Type type)
                 throw new ArgumentNullException(nameof(type));
             }
 
-            var actualType = type;
```
Comment (jezza323) [inline MicroLite/TypeConverters/TypeConverter.cs]: judging by the naming here, the intent was to resolve the actual type anyway

**Comment by TrevorPilley** [comment]
Could you send this to https://github.com/TrevorPilley/MicroLite/tree/6.3.x instead so that we can get 6.3.4 built?

**Comment by TrevorPilley** [comment]
I've done the fix and published [MicroLite 6.3.4](https://www.nuget.org/packages/MicroLite/6.3.4)

**Comment by jezza323** [comment]
Thanks Trevor, sorry, i hadnt checked my messages for a few days

**Pull request closed without merging**